app/src/main/java/org/hyperledger/besu/cli/options/storage/PathBasedExtraStorageOptions.java

```diff
@@ -69,11 +76,28 @@ public class PathBasedExtraStorageOptions
 
   @Option(
       names = {
-        TRIE_LOG_PRUNING_WINDOW_SIZE,
+        TRIE_LOG_PRUNING_BATCH_SIZE,
+        TRIE_LOG_PRUNING_WINDOW_SIZE // deprecated
       },
       description =
-          "The max number of blocks to load and prune trie logs for at startup. (default: ${DEFAULT-VALUE})")
-  private Integer trieLogPruningWindowSize = DEFAULT_TRIE_LOG_PRUNING_WINDOW_SIZE;
+          "The batch size for processing trie log pruning at startup. "
```

Comment:
```suggestion
          "The batch size used by the trie log pruner."
```

This batch size is used whenever pruning occurs, not just at start up.

Reply:
Good catch! You're right.

I was thinking too narrowly about the startup context. The batch size is indeed used whenever pruning occurs. I'll update to your suggested wording.